requirements/requirements.txt

```diff
@@ -2,6 +2,7 @@ appdirs==1.4.3
 attrs==19.1.0
 boto3==1.9.116
 botocore==1.12.116
+certifi==2019.3.9
```

Comment:
what is this?

Reply:
https://pypi.org/project/certifi/

Reply:
I was less asking that to understand the library personally, and more to check what it's being used for and if this dependency was intended to be added. It seems odd that this project would just need to start supporting SSL now.

Reply:
This was intentionally added for the elasticsearch etl script to support planned changes in the Elsticsearch architecture.